src/ensembl/src/content/app/species-selector/state/speciesSelectorActions.ts

```diff
@@ -312,7 +267,13 @@ export const deleteSpeciesAndSave: ActionCreator<
     return true;
   });
 
-  dispatch(deleteSpecies(speciesName, committedSpecies));
+  analyticsTracking.trackEvent({
+    category: categories.SELECTED_SPECIES,
+    label: speciesName,
+    action: 'unselected'
```

Comment:
the doc says, unselect rather than unselected

Reply:
Updated it 👍